server/web/templates/proposals/submissions-list.handlebars

```diff
@@ -30,19 +30,88 @@
                                 <div class="col-md-2">
                                     <div class="btn-group float-right" role="group">
                                         {{#ifEquals feasibilityStatus 'Pending'}}
-                                            <button type="button" class="btn btn-success btn-sm" onclick="updateFeasibilityStatus('{{../_id}}', true)">
-                                                Approve
+                                            {{!-- <button type="button" class="btn btn-success btn-sm" onclick="updateFeasibilityStatus('{{../_id}}', true)"> --}}
+                                            <button type="button" class="btn btn-success btn-sm" onclick="openFeasibilityModal('{{../_id}}')">
+                                                Feasibility Checked
                                             </button>
+                                        {{/ifEquals}}
+                                        {{#ifNotEquals feasibilityStatus 'Revise Requested'}}
                                             <button type="button" class="btn btn-danger btn-sm" onclick="updateFeasibilityStatus('{{../_id}}', false)">
-                                                Reject
+                                                Revise Requested
                                             </button>
-                                        {{/ifEquals}}
+                                            {{/ifNotEquals}}
+
+                                            {{#ifEquals feasibilityStatus 'Revise Requested'}}
+                                             <button type="button" class="btn btn-danger btn-sm" onclick="updateFeasibilityStatus('{{../_id}}', false)" disabled>
+                                                Revise Requested
+                                            </button>
+                                            {{/ifEquals}}
```

Comment:
This is buggy and redundant! I'm logged in as chair and see this button. only coordinators are supposed to see this button.